### ISSUE TITLE
PT019 should not recommend usefixtures for parametrize values

### ISSUE THREAD

**Issue by Hawk777** [astral-sh/ruff#17599]
### Summary

Consider the following code:
```python
import pytest

@pytest.mark.parametrize("_foo", [1, 2, 3])
def test_thingy(_foo):
    print("hello")
```

When I run `ruff check --select=PT019` on it, it fails. However, according to a pytest dev replying to my question [here](https://github.com/pytest-dev/pytest/issues/13349), a value injected via `parametrize` is a fixture only due to an implementation detail that should not be relied upon. Therefore, it is inappropriate to use `@pytest.mark.usefixture` on it.

### Version

ruff 0.11.6

**Comment by MichaReiser** [comment]
The PT019 rule assumes that every parameter starting with a `_` is a fixture. It seems reasonable to exclude parameters for which a named `parametrize` exists.
